### PR TITLE
Feature name cleanup

### PR DESCRIPTION
**Pull request by NottyCode**
Updates to make the feature names consistent. Some were versioned, some were not, also sometimes we use MicroProfile and sometimes Micro Profile so this makes us use consistent name.

Also updated the descriptions for MicroProfile features to read Eclipse MicroProfile.

### CODE REVIEW

**Comment by NottyCode** [comment]
#build me please Libby

File: dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenAPI-1.0/resources/l10n/com.ibm.websphere.appserver.mpOpenAPI-1.0.properties [hunk 1]
```diff
@@ -14,4 +14,4 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature provides support for the MicroProfile OpenAPI for enterprise Java.
+description=This feature provides support for the Eclipse MicroProfile OpenAPI for enterprise Java.
```
Comment (kwsutter) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mpOpenAPI-1.0/resources/l10n/com.ibm.websphere.appserver.mpOpenAPI-1.0.properties]: Did we want to add "specification" after OpenAPI to be consistent with the other updates?

File: dev/com.ibm.websphere.appserver.features/visibility/public/jwtSso-1.0/com.ibm.websphere.appserver.jwtSso-1.0.feature [hunk 2]
```diff
@@ -5,7 +5,7 @@ singleton=true
 IBM-ShortName: jwtSso-1.0
 # we don't need servlet 4.0, but we specify it to suppress
 # a CWWKF0001E from jwt's use of servlet 3.0
-Subsystem-Name: JwtSso
+Subsystem-Name: Jwt SSO 1.0
```
Comment (mbroz2) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jwtSso-1.0/com.ibm.websphere.appserver.jwtSso-1.0.feature]: my vote is still for "JwtSso -> JSON Web Token Single Sing-On 1.0" over "Jwt SSO 1.0" or at least "JWT SSO 1.0"

File: dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature [hunk 3]
```diff
@@ -3,7 +3,7 @@ symbolicName=com.ibm.websphere.appserver.jwt-1.0
 visibility=public
 IBM-ShortName: jwt-1.0
 IBM-API-Package: com.ibm.websphere.security.jwt; type="ibm-api"
-Subsystem-Name: Json Web Token
+Subsystem-Name: Json Web Token 1.0
```
Comment (mbroz2) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jwt-1.0/com.ibm.websphere.appserver.jwt-1.0.feature]: Json -> JSON

**Comment by mbroz2** [comment]
I think we're also missing:
OSGi Debug Console -> OSGi Debug Console 1.0

**Comment by lzdanski** [comment]
Not much to review for ID, but lgtm. Adding ID reviewed label 👍

**Pull request merged**